src/sources/datadog/agent/logs.rs

```diff
@@ -105,7 +105,7 @@ pub(crate) fn decode_log_body(
                 Ok(Some((events, _byte_size))) => {
                     for mut event in events {
                         if let Event::Log(ref mut log) = event {
-                            log.try_insert(path!("status"), status.clone());
+                            // log.try_insert(path!("status"), status.clone());
```

Comment:
Can we leave a comment on why this is removed, or a TODO around what we'd end up doing when it's resolved in the agent or logs backend?

Reply:
Ah, yeah, I meant to leave a comment but forgot. Thanks for catching that.

Reply:
I would generally prefer not to leave commented-out code in the project. If we aren't going to be collecting `status` -- and we need to clear that its removal doesn't require a deprecation cycle -- then we should nix the line entirely.